data_dir/disk_conf/hybrid_raid.py

```diff
@@ -0,0 +1,784 @@
+#!/usr/bin/env python3
+"""
+This script ensures that the system is in a state in which scylla can successfully start.
+Such logic includes:
+  * Verifying the filesystem on top of NVMe RAID and/or a single pd-ssd
+  * Re-configuring mounts to match the state of the system
+  * Configuring scylla to replace itself if this node lost its data
+"""
+import argparse
```

Comment:
@yarongilor where this script is coming from ?

Reply:
https://github.com/scylladb/scylla-qa-internal/pull/27